Makefile

```diff
@@ -1,8 +1,13 @@
+TAGS="dev"
+
 run:
 	go run ./examples/host
 
 build:
 	go build ./examples/host
 
+build/tags:
+	go build -tags ${TAGS} ./examples/host

```

Comment:
Does this means that we no longer library and we will provide some form of CLI as part of release?

Reply:
I would say still a library with an optional cli if a developer wants it.